backend/src/test/resources/application.properties

```
@@ -0,0 +1,6 @@
+server.port=${PORT:8080}
+spring.data.mongodb.uri=${MONGODB_URI:mongodb://localhost/wilico?retryWrites=true&w=majority}
+
+
+security.secret=123
+security.admin.name=123
```

I would at least set the secret via an environment variable
```suggestion
security.secret={SECURITY_SECRET}
security.admin.name={SECURITY_ADMIN_NAME}
```